plugins/Kaleidoscope-MouseKeys/src/kaleidoscope/plugin/MouseKeys.cpp

```diff
@@ -92,31 +93,27 @@ EventHandlerResult MouseKeys::onNameQuery() {
 
 // -----------------------------------------------------------------------------
 EventHandlerResult MouseKeys::onSetup(void) {
-  kaleidoscope::Runtime.hid().mouse().setup();
```

Comment:
Because I didn't change anything functional about how the absolute mouse works in this PR.  I knew the above line wasn't necessary, but I wasn't sure about the other `setup()` call.